main.go

```diff
@@ -0,0 +1,229 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"fmt"
+	"log/slog"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+
+	"github.com/charmbracelet/glamour"
+)
+
+// models
+var geminiModels = []string{
+	"gemini-2.0-flash-thinking-exp-01-21",
+}
+
+func main() {
+	// non interactive execution when query is specified on the command line.
+	queryFromCmd := flag.String("query", "", "query for the agent")
+	maxIterations := flag.Int("max-iterations", 20, "maximum number of iterations")
+	kubeconfig := flag.String("kubeconfig", "/Users/sunilarora/.kube/auto-ai-1", "path to the kubeconfig file")
```

Comment:
choose a different default?

Reply:
Defaulted to empty value. This will lead to default `kubectl` behavior.